### ISSUE TITLE
Windows: Conemu's own DLL injection breaks that of tup

### ISSUE THREAD

**Issue by mitsukuri** [gittup/tup#419]
Hello!

I've read about issues with running **tup** under **Conemu** on Windows, and I think I've found the culprit.

By default Conemu injects its own DLL named `ConEmuHk.dll`, and that seems to cause the `tup error: failed to inject dll: No such file or directory`.

As soon as I disabled injection of `ConEmuHk.dll` in Conemu's preferences, tup started working just fine.

If that reproduces, I think some notice would be nice to have added to documentation somewhere.

**Comment by gittup** [comment]
Thanks for the report!

I'm having trouble reproducing this. I downloaded ConEmu 210314 [64] {Preview} and I have "Inject ConEmuHk.dll into all processes" checked in settings. Compiling some .c files with tup works fine (I tried both the default cmd shell and the Bash::CygWin bash ones). Any idea what I'm missing?

**Comment by mitsukuri** [comment]
Hmmm... I was using 210206 [32] as of initial report, now I've tried 210314 [64] to same effect... I've really no clue why it happens on my side at the moment.

**Comment by gittup** [comment]
I was able to reproduce it by trying to spawn 'touch' instead of 'gcc'. I added a check if tup is running with ConEmuHk.dll enabled, and it prints out an error message indicating that things won't work as expected. I also took a look at the ConEmu source to see if there's an easy way to temporarily disable ConEmuHk.dll injection (like an environment variable setting that tup could set), but I didn't see anything obvious.

@Maximus5, is there anything tup could do to disable the injection temporarily for processes that it spawns? Or is an error message the best I can do here?
